### ISSUE TITLE
Suggestion: make jumps consume "stamina" instead of "calories"

### ISSUE THREAD

**Issue by d4n13lus-com** [DigitalzombieTLD/JumpMod#1]
Hi, thanks for your work, first!

Can you make a version of the mod that consumes stamina instead of calories?
Seems more realistic imho

and if possible, configurable in mod menu options would be perfect! 10% stamina per jump, 5%, 20%... 

regards!

**Comment by DigitalzombieTLD** [comment]
Hey, sorry for the late reply. Always forget checking here ... nice idea. I think thats possible :)

**Comment by mikeloeven** [comment]
Agreed the calorie consumption for jumping is a bit off take it froms someone trying to loose a bit of weight I WISH simply jumping burned 15 calories in real life ...
